modules/afsocket/afsocket-source.c

```diff
@@ -399,10 +399,16 @@ afsocket_sd_process_connection(AFSocketSourceDriver *self, GSockAddr *client_add
 
   if (self->num_connections >= self->max_connections)
     {
+      gchar *source_name = cfg_tree_get_rule_name(&self->super.super.super.cfg->tree, ENC_SOURCE, self->super.super.super.expr_node);
+
       msg_error("Number of allowed concurrent connections reached, rejecting connection",
                 evt_tag_str("client", g_sockaddr_format(client_addr, buf, sizeof(buf), GSA_FULL)),
                 evt_tag_str("local", g_sockaddr_format(local_addr, buf2, sizeof(buf2), GSA_FULL)),
+                evt_tag_str("source_name", source_name),
```

Comment:
The location tag should define exactly which LogPipe it deals with, sure sometimes a calculation due to buffers needed (that should be improved...). Still the source name could be not better when it is `anon-`, I would suggest to print the stats name here, as that's the data you want to cross-check anyway.

Reply:
Thank you for pointing this out. I discussed the topic with the support team. They prefer keeping the current version. Together with the configuration element location it seems good enough for the current purpose.